libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -238,7 +238,7 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
 
     // @Param: SYNAIRSPEED
     // @DisplayName: Enable the use of synthetic airspeed
-    // @Description: This enable the use of synthetic airspeed for aircraft that don't have a real airspeed sensor. This is useful for development testing where the user is aware of the considerable limitations of the synthetic airspeed system, such as very poor estimates when a wind estimate is not accurate. Do not enable this option unless you fully understand the limitations of a synthetic airspeed estimate.
+    // @Description: This enables the use of synthetic airspeed in TECS for aircraft that don't have a real airspeed sensor. This is useful for development testing where the user is aware of the considerable limitations of the synthetic airspeed system, such as very poor estimates when a wind estimate is not accurate. Do not enable this option unless you fully understand the limitations of a synthetic airspeed estimate. This option has no effect if a healthy airspeed sensor is being used for airspeed measurements.
```

Comment:
synthetic airspeed is being computed continuously in the DCM and EKF filters..it could be needed at anytime.....this is an advanced user parameter...we dont put every bit of possibly related information in the parameter metadata...for standard user params the wiki should explain how to use them...advanced user params do not have detailed explanations in the wiki as a rule...sometimes yes, sometimes no..and there are a LOT of advance user parameters...these usually require the user to be familiar with the code, or get helpunderstanding how to use them